bin/admin-kubeconfig.bash

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
```

Comment:
Why must this be a separate file from `user-kubeconfig`? Shouldn't the config look the same in the end?

Reply:
The resulting kubeconfig is almost exactly the same, with a few minor differences (such as calling the user `admin@$cluster_name` instead of `user@$cluster_name` and giving the kubeconfig a different filename). The most significant difference between the two scripts is that `admin-kubeconfig.bash` should be able to be used on any cluster whereas `user-kubeconfig.bash` only applies to the workload cluster, so `admin-kubeconfig.bash` needs some arguments specifying the cluster while `user-kubeconfig.bash` does not. I guess a solution to reducing the duplicated code would be to consolidate the shared code in `common.bash`.

Reply:
Right, well then I would rather like a `kubeconfig.bash` file that have these small difference by giving it an argument or something like that. I feel like `common.bash` only should have stuff that is common for all scripts.

Reply:
Sure, that makes sense. I'd be fine with doing it that way.